src/main/java/duke/helper/Storage.java

```diff
@@ -0,0 +1,128 @@
+package duke.helper;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.DukeException;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+/**
+ * Storage is the local class that deals with the logic for storing and updating tasks data locally
+ */
+
```

Comment:
Very large space here! Consider cutting down on it to make things more readable.